appveyor.yml

```diff
@@ -14,15 +14,27 @@ install:
   - cmd: SET PATH=C:\sbt\sbt\bin;%JAVA_HOME%\bin;%PATH%
   - cmd: SET SBT_OPTS=-XX:MaxPermSize=2g -Xmx4g
```

Comment:
Good to know! I've been trying to stick to sbt 0.13.8 for now though, to be sure sbt-coursier is fine with it.